### DISCUSSION TITLE
Does it has feature to make it frameless?

### DISCUSSION THREAD

**Discussion by JenuelDev** [neutralinojs/neutralinojs#939] [Q&A]
is it possible to make it frameless so that I can use my own appbar?

**Comment by pathange-s** [comment]
I don't think there is one. Check [here](https://github.com/neutralinojs/neutralinojs/issues/507).

**Comment by hv6erT** [comment]
Of course it is. You can set borderless to true in neutralino.config.json. Your app would be renderered without system titlebar and window controlls.
